typings/index.d.ts

```diff
@@ -1670,6 +1670,8 @@ export class TextChannel extends TextBasedChannel(GuildChannel) {
   public fetchWebhooks(): Promise<Collection<Snowflake, Webhook>>;
 }
 
+export type TextChannelResolvable = Snowflake | TextChannel;
```

Comment:
I'm pretty sure types have to go to the bottom, please put it alongside `ChannelResolvable`, and same for the other types.

Reply:
Ah, thank you for pointing that out! I do see a region for type definitions now... but I also see a bunch that are not below this line, but that's out of scope here. I'll just move the ones I've made for now, and perhaps the others can be tidied up separately.